pkg/R/RDD.R

```diff
@@ -64,43 +80,62 @@ setMethod("initialize", "PipelinedRDD", function(.Object, prev, func, jrdd_val)
     .Object@prev_jrdd <- getJRDD(prev)
     # Since this is the first step in the pipeline, the prev_serialized
     # is same as serialized here.
-    .Object@env$prev_serialized <- .Object@env$serialized
+    .Object@env$prev_serializedMode <- prev@env$serializedMode
```

Comment:
With the new change, these two lines of comments are stale and should be removed